.github/workflows/labeling.yml

```diff
@@ -0,0 +1,24 @@
+# This workflow adds the 'ci-passed' label to a pull request once the 'CI Check' workflow completes successfully.
```

Comment:
How about doing this in `.github/workflows/ci-checks.yml`?
You can move the `add_ci_passed_label` and declare that it depends on `check_ci_status`:

```yaml
  add_ci_passed_label:
    needs: [check_ci_status]
```
Then you can remove `if: github.event.workflow_run.conclusion == 'success'`.